frontend/src/ol/staticLayers/layerTypes/group.js

```diff
@@ -7,8 +7,17 @@ const groupLayer = (layerData, handleIsLoading) => {
     title: layerData.title,
     type: layerData.layerType,
     legend: layerData.legend,
-    layers: layerData.layers.map((layer) => staticLayerGenerator({ ...layer, visible: true }, handleIsLoading)),
-    timeseries: layerData.timeseries,
```

Comment:
should be still compatible with this data structure

Reply:
I fixed this and tested it. Please check it. I removed 'timeseries' key from documentation (moved it to layerOptions). Should I also leave the documentation as it was before + add layerOptions description?